test/functional/severity-control.test.js

```diff
@@ -6,10 +6,9 @@ const dir = fixtures + '/severity-control';
 const fs = require('fs');
 let vulns = {};
 
-tap.beforeEach(function (done) {
+tap.beforeEach(function () {
   // only contains medium + low - this file is read using fs to ensure refresh
   vulns = JSON.parse(fs.readFileSync(dir + '/vulns.json', 'utf8'));
-  done();
```

Comment:
If the function does not return a promise, then it is now run synchronously ([docs](https://github.com/tapjs/node-tap/blob/main/docs/src/content/docs/api/test-lifecycle-events/index.md#tbeforeeachfnchildtest))